internal/functions/serve/templates/main.ts

```diff
@@ -2,6 +2,7 @@ import {
   STATUS_CODE,
   STATUS_TEXT,
 } from "https://deno.land/std/http/status.ts";
+import * as posix from "https://deno.land/std/path/posix/mod.ts";
```

Comment:
can use `path/mod.ts` right? doesn't need `posix`

Reply:
`path/posix` imports fewer dependencies, ie. excluding path/windows which is never needed in our containerized runtime.